source/guide_focalboard.rst

```diff
@@ -0,0 +1,98 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: web
+.. tag:: project-management
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/focalboard.svg
+      :align: center
+
+##########
+Focalboard
+##########
+
+.. tag_list::
+
+Focalboard_ is an open source, self-hosted alternative to Trello, Notion, and Asana. 
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+Download the latest release_ to your home directory and extract the application.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ wget https://github.com/mattermost/focalboard/releases/download/v0.6.1/focalboard-linux.tar.gz
+ [isabell@stardust ~]$ tar xzfv focalboard-linux.tar.gz
```

Comment:
Shouldn’t it be with a `-`?
```
  [isabell@stardust ~]$ tar -xzfv focalboard-linux.tar.gz
```

And according to the [style guide](https://github.com/Uberspace/lab/blob/main/STYLE.md) one should use extend flags, so the following would be even better (there is no extended flag for `-x` “extract”):
```suggestion
 [isabell@stardust ~]$ tar -x --gzip --file focalboard-linux.tar.gz
```

I am intentionally omitting `--verbose` as it is not required.

Reply:
And we should include instructions to remove the Archive after extracting it:

```
The extraction step unpacked the content, so you can clean up and remove the archive:

 .. code-block:: console

   [isabell@stardust ~]$ rm focalboard-linux.tar.gz
   [isabell@stardust ~]$
```